src/patternfly/components/List/docs/code.md

```diff
@@ -1,11 +1,11 @@
 ## Overview
 
-Typically used with a modifier class.
+The list component should be used with a modifier to determine its style as a `<ul>` or `<ol>`. `.pf-c-list__item` is required. Lists can be nested up to any level.
```

Comment:
I think you can just remove the whole overview, doesn't look like any of this is necessary anymore.